graphql-dgs/src/test/kotlin/com/netflix/graphql/dgs/DgsSchemaProviderTest.kt

```diff
@@ -522,6 +522,143 @@ internal class DgsSchemaProviderTest {
         assertThat(provider.dataFetcherInstrumentationEnabled["Show.title"]).isFalse
     }
 
```

Comment:
Should we add a test for the subscription annotation as well?

Reply:
good point, added